src/pages/api/modules.ts

```diff
@@ -0,0 +1,31 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import data from "../../../data.json";
```

Comment:
Double quotes!?

Reply:
Yah, eslint config has prettier setup which is doubles by default iirc